frontend/src/components/StationMiniCard/StationMiniCard.scss

```diff
@@ -0,0 +1,52 @@
+@import "../../variable.scss";
+
+.station_mc {
+  box-sizing: border-box;
+  height: 5rem;
+  margin-bottom: 1.5rem;
+  background-color: $colorLight;
+  border: $borderDark;
+  border-radius: 10px;
+  box-shadow: $boxShadow_inv;
+  display: grid;
+  grid-template-columns: 25% 35% 1fr 15%;
+  grid-template-rows: 1fr 55%;
+  align-items: center;
+}
+
+.station_mc_logo {
+  width: 2.5rem;
+  height: 2.5rem;
+  padding: 0.25rem 0.4rem;
+  background: $gradient90deg;
+  border-radius: 10px;
+  box-shadow: $boxShadow_inv;
+  grid-area: 1 / 1 / 3 / 2;
+  justify-self: center;
+}
+
+.plug-type_logo {
+  width: 2.2rem;
```

Comment:
Idem

Reply:
this value is fixed on purpose, rem is the good unit here.